sunpy/spectra/spectrogram.py

```diff
@@ -9,6 +9,7 @@
 from random import randint
 from copy import copy
 from math import floor
+from named_constants import Constants
```

Comment:
Where is named_constants from?  Is it this - https://pypi.python.org/pypi/named_constants/1.0 ? The PR is failing because this package is not a SunPy dependency, and is not required to fix this issue.